### ISSUE TITLE
v2.4.1: getAvailableApps () always returns empty Array

### ISSUE THREAD

**Issue by skizzo** [leanmotherfuckers/react-native-map-link#87]
Hi,
the `showLocation()` function stopped working for me, even when testing it on a real iOS device and not the simulator. So I went through the code of the latest version (`2.4.1` at the time of writing this), and noticed this in **`src/utils.js`**:

```
export const getAvailableApps = async (prefixes) => {
  let availableApps = []
  for (let app in prefixes) { // prefixes is always undefined -> no apps will be returned
    if (!prefixes.hasOwnProperty(app)) {
      continue
    }
    let avail = await isAppInstalled(app, prefixes)
    if (avail) {
      availableApps.push(app)
    }
  }

  return availableApps
}

// ...

export function askAppChoice({ dialogTitle, dialogMessage, cancelText, appsWhiteList }) {
  return new Promise(async (resolve) => {
    let availableApps = await getAvailableApps() // shouldn't this be called with a parameter?
    // ...
  })
}
```

Why is the `prefixes` parameter not handed to the `getAvailableApps ()` function? Or, differently asked, **can v2.4.1 work for anybody at all**?

Thanks for some infos.

**Comment by tschoffelen** [comment]
I just opened #87 with a possible fix, but don't have time to actually run it and check if it works. Could you have a look?

**Comment by skizzo** [comment]
Wow thanks, that was fast 👍

Your commit fixes it, the ActionSheet's showing again, everything's alright. Might want to push this soon into your release since this bug's kind of a show stopper :D Thx!

**Comment by tschoffelen** [comment]
Released to NPM as `react-native-map-link@2.4.3`.
